docs/pages/guide/02-start.md

```diff
@@ -0,0 +1,409 @@
+| <span class="c40">H2</span>     | <span class="c22">Now</span>          |
+| <span class="c40">Hive</span>   | <span class="c22">Now</span>          |
+| <span class="c40">Presto</span> | <span class="c22">Now</span>          |
+| <span class="c33">MySQL</span>  | <span class="c22">Now</span>          |
+{:.table}
+
+More information on dialects and how to use them can be found at: <span
+class="c13"><a href="https://www.google.com/url?q=https://elide.io/pages/guide/v5/04-analytics.html%23dialects&amp;sa=D&amp;ust=1606228594025000&amp;usg=AOvVaw3kCKuvbnXUCJCPe8SCcGF7" class="c14">https://elide.io/pages/guide/v5/04-analytics.html#dialects</a></span><span
+class="c11"> </span>
+
+<span class="c29">About data models</span>
+------------------------------------------
+
+A semantic model is a simplified representation of the data that resides
+in your database.  Semantic models consist of concepts like Tables,
+Dimensions, and Measures.  While the physical database schema (The
+schema describes the data, how it is organized and managed in the
+physical data source) can be complex, the semantic model is flat (there
+are no relationships between tables) and the columns are referenced by
+simple names (as opposed to complex formulas or derivations). A <span
+class="c40 c48">data model</span><span class="c11"> is an abstraction
+which describes how you can use the data once it has been loaded. Note
+that the model need not be a one to one match to the schema in the Data
+source! While models can correspond to your underlying data store schema
+as a one-to-one representation, Yavin does not require this and the
+operating model of Yavin encourages you to think of data models as being
+isolated on a per service basis. We recommend creating data models that
+only support precisely the bits of data you need from your underlying
+schema and no more!  A design that emphasizes isolation makes Yavin an
+effective tool to reduce interdependency among services and maximizing
+the separation of reporting, from physical data design. </span>
+
+<span class="c11"></span>
+
+Aspects of the data source you can configure include <span
+class="c40">Tables</span>, <span class="c40">Dimensions</span> and <span
+class="c40">Measures.</span>
+
+More  information on the Elide Analytics query support and data
+modelling can be found at this URL: <span
+class="c13"><a href="https://www.google.com/url?q=https://elide.io/pages/guide/v5/04-analytics.html%23overview&amp;sa=D&amp;ust=1606228594027000&amp;usg=AOvVaw2q1sp9bx_xpv3bXaEJGwG4" class="c14">https://elide.io/pages/guide/v5/04-analytics.html#overview</a></span><span
+class="c11"> </span>
+
+### <span class="c9">Writing your first data model</span>
+
+The “<span
+class="c13"><a href="https://www.google.com/url?q=https://elide.io/pages/guide/v5/04-analytics.html%23model-configuration&amp;sa=D&amp;ust=1606228594028000&amp;usg=AOvVaw1vAW2xdLnJnsUQnKsE9jO5" class="c14">Getting Started Docs</a></span>”
```

Comment:
Sentence structure here is awkward.   Also, the link doesn't point to the getting started document.  It is the section on the analytic API.  Maybe just remove this sentence since you provide links later on as well.